core/java/iesi-rest-without-microservices/src/test/java/io/metadew/iesi/server/rest/configuration/security/jwt/ComponentsControllerSecurityTest.java

```diff
@@ -96,15 +108,19 @@ void testGetAllNoUser() throws Exception {
                     "DATASETS_READ@PUBLIC",
                     "DATASETS_WRITE@PUBLIC"})
     void testGetAllNoComponentReadPrivilege() throws Exception {
-        assertThatThrownBy(() -> componentsController.getAll())
+        Pageable pageable = Pageable.unpaged();
+        assertThatThrownBy(() -> componentsController.getAll(pageable, null))
                 .isInstanceOf(AccessDeniedException.class);
     }
 
```

Comment:
could you add some more tests? e.g. filtering, sorting, pagination

Reply:
I think the tests should be added to a new class 'ComponentDtoRepositoryTest'. Here you should really test the queries being excuted on a dummy database